### PR TITLE
Extended Material Ui theme, Made Componets and Forgot Password Page

### PR DESCRIPTION
**Pull request by smyaseen**
## Goal:

To Extend material UI theme.

## Work: 

Extended theme, made components like app Drawer, app bar, card, container, text field, customized button, added global styles, made forgot password page.
 
 ## Trello Link:
 
 https://trello.com/c/8LB0wzap

### CODE REVIEW

File: src/Pages/ForgetPassword.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <Grid alignItems="center" container direction="column" justify="center" style={{ minHeight: '100vh' }}>
+        <Typography variant="h1">Reset Your Password</Typography>
+        <br />
+        <Card margin="auto" maxWidth={400} minWidth={300}>
+          <div style={{ textAlign: 'center' }}>
+            <Typography color="secondary" variant="h3">
+              Forgot Password ?
+            </Typography>
+            <br />
+            <Typography color="primary" paragraph style={{ wordWrap: 'break-word' }}>
```
Comment (SiddiquiMonis401) [inline src/Pages/ForgetPassword.js:20]: Avoid inline styling

File: src/components/Drawer.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      <AppBar
+        className={clsx(classes.appBar, {
+          [classes.appBarShift]: open,
+        })}
+        position="fixed"
+      >
+        <Toolbar>
+          <IconButton
+            aria-label="open drawer"
+            className={clsx(classes.menuButton, {
```
Comment (ShahirZain) [inline src/components/Drawer.js:124]: can't we destructure the classes at the top and then use it?

File: src/components/TextField.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/core/styles';
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+export default function BasicTextFields({ variant, width, type, label }) {
+  const useStyles = makeStyles((theme) => ({
```
Comment (ShahirZain) [inline src/components/TextField.js:5]: we can use styled component to give custom styling instead of makeStyles

**Review by ShahirZain — CHANGES_REQUESTED** [review]

File: src/components/Drawer.js (hunk 2 above)
Comment (smyaseen) [reply src/components/Drawer.js:124]: We can I think, it was default on material UI documentation, so I didn't change it.

Comment (smyaseen) [reply src/components/Drawer.js:124]: Destructured.

File: src/components/Drawer.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
+import { Grid, MenuItem } from '@material-ui/core';
+import { PersonRounded } from '@material-ui/icons';
+import { makeStyles, useTheme, withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
```
Comment (SiddiquiMonis401) [inline src/components/Drawer.js:5]: I think tree shaking is enabled in default webpack bundle. So, no need to export each component separately ..

**Pull request merged**